packages/aws-cdk-lib/aws-events/lib/event-bus.ts

```diff
@@ -78,6 +79,15 @@ export interface EventBusProps {
    * @default - no partner event source
    */
   readonly eventSourceName?: string;
+
+  /**
+   * Dead Letter Queue for the event bus
```

Comment:
Looks like the docs have this as `dead-letter queue`. Can you change this comment to `Dead-letter queue` ?

https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-rule-event-delivery.html#eb-rule-dlq